pandas/core/generic.py

```diff
@@ -10210,6 +10210,44 @@ def _doc_parms(cls):
 nan
 """
 
+_max_examples = """\
+Examples
+--------
+``MultiIndex`` series example of monthly rainfall
+
+>>> index = [np.tile(['London', 'New York'], 3),
+...          np.repeat(['Jun', 'Jul', 'Aug'], 2)]
+>>> s = pd.Series([47, 112, 35, 117, 54, 113], index=index)
+>>> s.rename_axis(['city', 'month'], inplace=True)
+>>> s
+city      month
+London    Jun       47
+New York  Jun      112
+London    Jul       35
+New York  Jul      117
+London    Aug       54
+New York  Aug      113
+dtype: int64
+
+>>> s.max()
+117
+
+Sum using level names, as well as indices
```

Comment:
Think you mean to say `Max` here and not `Sum`

Reply:
My bad, good catch!